source/configuration_manual/authentication/password_schemes.rst

```diff
@@ -104,7 +104,7 @@ setting in ``dovecot-sql.conf.ext``
 :ref:`authentication-passwd_file` : CRYPT is used
 by default, but can be changed with ``scheme`` parameter in passdb args.
 
-:ref:`authentication-passwd`, :ref:`authentication-shadow`, ``vPopMail``: CRYPT is used by default
+:ref:`authentication-passwd`, ``vPopMail``: CRYPT is used by default
```

Comment:
vpopmail has already been removed in 2.3.13

Reply:
I removed all vpopmail references in a separate commit.